x/group/server/msg_server.go

```diff
@@ -238,6 +253,8 @@ func (s serverImpl) CreateGroupAccount(ctx types.Context, req *group.MsgCreateGr
 		return nil, err
 	}
 
+	// TODO Once we update to use ADR 028, we'll also need to make sure
+	// that there is a module account created in x/auth (#211).
```

Comment:
I'm thinking maybe as part of ADR 033 we actually had some sort of method `EnsureModuleAccountExists()` because this sort of functionality will be needed generically by ADR 033 users and we don't necessarily need to introduce an `x/auth` dependency. Wdyt @blushi ?

Reply:
Sounds good, I've updated the comment to not mention `x/auth` in particular but ADR 033.